### ISSUE TITLE
[Question] Debounce email

### ISSUE THREAD

**Issue by derencel20** [nodemailer/nodemailer#1129]
Is there a native feature for debounce in node-mailer? I can't find one in the documents.

Our App is multi-tenant and uses Kubernetes for cluster so I think we couldn't debounce this on nodejs level.
**Stack**
- Node.js
- Express
- Kubernetes (Cluster)
- Kafka

So here's the scenario.
We have our users in messaging center. User A sends a message to User B. After sending the message, we'll have to send an email notifying User B that he has a new message in his inbox so that he would read it. If User A sends multiple message to User B, it would also send multiple emails to User B. And we wouldn't want that. So, we would like to have User B receive one email only for User A sending multiple message to User B in a span of 10 mins.

**User A => User B conversation**
User A: Hello
User A: I would like to propose this change

**Backend**
- would send one email only after user A types in 'Hello' in the span of 10 mins regardless of the message sent to the user

**Comment by derencel20** [comment]
@andris9 could this be a feature request if it's not implemented yet?

**Comment by andris9** [comment]
This is out of scope for Nodemailer as Nodemailer is a lower level email sending library, it does not handle any application logic, you have to do this yourself in your application code.
